content/en/tracing/setup/dotnet-core.md

```diff
@@ -271,7 +244,9 @@ For example, on Windows:
 ```cmd
 rem Set environment variables
 SET DD_TRACE_AGENT_URL=http://localhost:8126
-SET DD_SERVICE_NAME=MyService
+SET DD_ENV=prod
+SET DD_SERVICE=MyService
+SET DD_VERSION=abc123
 SET DD_ADONET_ENABLED=false
```

Comment:
```suggestion
SET DD_ADONET_ENABLED=true
```

Reply:
Having `DD_ADONET_ENABLED` was just an example of how users can change the default values. All integration are already enabled by default. Feel free to remove the line entirely if it's confusing.